src/WatchDogs.FakeSource/IDataGeneratorRepeaterBackroundTask.cs

```diff
@@ -0,0 +1,8 @@
+﻿
+namespace WatchDogs.FakeSource;
+
+public interface IDataGeneratorRepeaterBackroundTask
+{
+    void Start();
```

Comment:
You are on a right track here, but please make it `async` and also make it accept optional `CancellationToken`

`Task StartAsync(CancellationToken token = default);`

Reply:
okey done.

Reply:
Done.